content/docs/ai/ai-intro.md

```diff
@@ -7,18 +7,119 @@ updatedOn: '2023-10-07T10:43:33.361Z'
 
-Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
+Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open-source database that you know and trust. There's no need for data migration or a 3rd-party vector storage solution.
 
-To get started, see [The pgvector extension](/docs/extensions/pgvector).
+Neon's AI Starter Kit provides resources, starter apps, and examples to help get you started.
 
-## Example applications
+<CTA title="Ship faster with Neon's AI Starter Kit" description="Sign up for Neon Postgres and get a headstart on your AI app.  Our starter apps and resources will help you get up and running qucikly." buttonText="Sign Up" buttonUrl="https://console.neon.tech/signup" />
 
-Check out the following AI application examples built with Neon.
+The **Neon AI Starter Kit** includes:
+
+- Neon Postgres with the latest version of the Postgres [pgvector](https://neon.tech/docs/extensions/pgvector) extension for storing vector embeddings
+- A variety of hackable, fully-featured [AI starter apps](#ai-starter-apps) to get you started:
+  - AI Chatbot
+  - RAG chatbot
+  - Semantic search
+  - Hybrid search
+  - Reverse image search
+  - Chat with PDF
+- A [vector search optimization guide](/docs/ai/ai-vector-search-optimization) for better AI application performance
+- A [scaling guide](/docs/ai/ai-scale-with-neon) for scaling your app with Neon's Autoscaling and Read Replica features
+- A collection of [AI apps built with Neon](#ai-apps-built-with-neon) that you can reference while building your own app
+
+## AI basics
+
+<DetailIconCards>
+<a href="/docs/ai/ai-concepts" description="The basics of building AI applications with Postgres" icon="openai">AI concepts</a>
+
+<a href="/docs/extensions/pgvector" description="Learn about the pgvector Postgres extension" icon="openai">The pgvector extension</a>
+
+</DetailIconCards>
+
+## Preparing your AI app for production
+
+<DetailIconCards>
+
+<a href="ai-vector-search-optimization" description="Optimize vector search for better application performance" icon="openai">Vector search optimization</a>
+
+<a href="/docs/ai/ai-scale-with-neon" description="Scale your app with Neon's Autoscaling, Read Replica, and serverless driver features" icon="openai">Neon AI scaling guide</a>
+
+</DetailIconCards>
+
+## AI starter apps
+
+Hackable, fully-featured, pre-built [starter apps](#ai-starter-apps) to get you up and running.
```

Comment:
presently, this hyperlink links it to the same section it's in.

how about we update it to point to https://github.com/neondatabase/examples/tree/main/ai